content/articles/razor-pages-web-app-tutorial-using-asp.net-core/index.md

```diff
@@ -0,0 +1,371 @@
+### Table of Contents
+
+[Introduction](#Introduction)
+
+[Prerequisites](#Prerequisites)
+
+[Creating a razor pages web application](#Creating-a-razor-pages-web-application)
+
+[Integrating a model into a razor pages app](#Integrating-a-model-into-a-razor-pages-app)
+
+[Updating/adding data to the razor pages web application](#Updating/adding-data-to-the-razor-pages-web-application)
+
+[Including a search bar](#Including-a-search-bar)
+
+[Including validation](#Including-validation)
+
+[Conclusion](#Conclusion)
+
```

Comment:
Please note that only the first word should be capitalized in a heading. Also, the "Table of contents" section usually goes under the "Introduction" section.
Lastly, the words after the '#' all need to be in lower case. Remove the white spaces in between.